spec/factories/company_values.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :company_value do
+    title { 'Random Title' }
```

Comment:
Try using names more or less in the following manner - it will later help you find that value on the page during tests (with 'Random Title' you can have a collision with other objects)

```suggestion
    title { 'Company Value Title' }
```